### ISSUE TITLE
Another registry key

### ISSUE THREAD

**Issue by AlekseyPopovv** [crow-translate/crow-translate#622]
Valid registry key:
HKEY_CURRENT_USER\Software\crow-translate\crow-translate

Missing registry key:
HKEY_CURRENT_USER\Software\Crow Translate\Crow Translate

**Comment by Shatur** [comment]
In the latest release we switched to ` HKEY_CURRENT_USER\Software\crow-translate\crow-translate`. You can remove the old one.

**Comment by AlekseyPopovv** [comment]
In version 2.11.0 the path in the registry is still HKEY_CURRENT_USER\Software\crow-translate\crow-translate

**Comment by Shatur** [comment]
Yes, and it will be like this from now on.

**Comment by AlekseyPopovv** [comment]
The installer does not remove this HKEY_CURRENT_USER\Software\crow-translate\crow-translate. Only removes this HKEY_CURRENT_USER\Software\Crow Translate\Crow Translate.
It needs to be fixed in the NSIS.

**Comment by Shatur** [comment]
We generate NSIS automatically. I think if you run the uninstaller from the previous version, it will remove `HKEY_CURRENT_USER\Software\Crow Translate\Crow Translate`. And after installing new version and running uninstaller it will remove `HKEY_CURRENT_USER\Software\crow-translate\crow-translate`. Correct?

**Comment by AlekseyPopovv** [comment]
No. The new version of the installer still contains HKEY_CURRENT_USER\Software\Crow Translate\Crow Translate

**Comment by Shatur** [comment]
Oh, I see. Probably it's because of this variable:
https://github.com/crow-translate/crow-translate/blob/bf47e3502a4edb8024a0d20030d58d6d164479a1/CMakeLists.txt#L18
Could you try changing it to `${PROJECT_NAME}` locally and see if it fixes the issue? I don't have Windows machine right now.

**Comment by AlekseyPopovv** [comment]
Can you add an option to save the settings file in another location?
For example:
"P:\Portable Soft\Crow Translate Portable\crow.exe" -inipath="P:\Portable Settings\Crow Translate Portable"

**Comment by Shatur** [comment]
> I don't understand this. What to do after the change?

Compile and run the installer. See instructions [here](https://github.com/crow-translate/crow-translate#building-a-package-using-cpack).

>  Do you speak Russian?

I do, but I prefer to speak English on English-speaking platforms.

> Can you add an option to save the settings file in another location?

This could be added, feel free to open a new issue about it.
